dotcom-rendering/package.json

```diff
@@ -64,7 +66,7 @@
     "@types/sanitize-html": "^1.27.1",
     "ajv": "^8.1.0",
     "ajv-formats": "^2.0.2",
-    "aws-sdk": "2.814.0",
+    "babel-plugin-prismjs": "^2.0.1",
```

Comment:
Oh this was recently removed [here](https://github.com/guardian/dotcom-rendering/pull/3825) - a rebase and fixing any merge conflicts should sort it.

Reply:
Fixed it, thanks Ravi!